draft-ietf-quic-qpack.md

```diff
@@ -550,18 +550,36 @@ the block.  Blocking decoders use the Largest Reference to determine when it is
 safe to process the rest of the block.
 
 `Base Index` is used to resolve references in the dynamic table as described in
-{{indexing}}.  To save space, Base Index is encoded relative to Largest
-Reference using a one-bit sign flag.
+{{indexing}}.
 
-    baseIndex = largestReference + deltaBaseIndex
+To save space, Base Index is encoded as a relative to the Base Index using a
```

Comment:
relative to Largest Reference

Reply:
Also, "encoded relative to" rather than "encoded as a relative to".  Otherwise, I need to see how we encode second cousins....  😉